eZ/Publish/API/Repository/Values/Content/Query/Criterion/Operator.php

```diff
@@ -24,4 +24,22 @@
     const BETWEEN = 'between';
     const LIKE = 'like';
     const CONTAINS = 'contains';
+
+    /**
+     * @return array
+     */
+    public static function getAvaliableOperators()
```

Comment:
Documentation referes to those names:
https://doc.ez.no/display/EZP/Criteria+reference

It does not say to use "=", ">=", etc.